src/components/icons/dateIcon.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+const DateIcon = props => (
```

Comment:
Not really an issue, but just wondering why make the svgs components? You can import them and use as <img> elements, or if you need inline svg for a specific reason you can import with an svg-react-loader ?

Reply:
It's more flexible if you need to modify them for new use-cases one day (this has often happened for me in previous projects. Now i just make all my SVGs react components by default. it doesn't hurt.)